src/components/PortForwarding.vue

```diff
@@ -81,10 +112,18 @@ export default {
       }));
     },
   },
+  mounted() {
+    ipcRenderer.on('k8s-check-state', (event, stt) => {
+      this.$data.k8sState = stt;
+    });
+  },
   methods: {
     update(state, service) {
       ipcRenderer.invoke('service-forward', service, state);
     },
+    handleCheckbox(value) {
+      this.$emit('change', value);
```

Comment:
`change` seems a bit too generic an event for specifically "toggled a filter"; this would make more sense if 1) we fired this on every port addition / removal, and 2) didn't pass a value with the event.  Alternatively, we could rename the event to indicate that it only filters Kubernetes internal services.

Reply:
ok